install/resources/upgrade50.php

```diff
@@ -35,3 +35,22 @@ function upgrade50_dbchanges()
 	$output->print_contents("<p>Click next to continue with the upgrade process.</p>");
 	$output->print_footer("50_done");
 }
+
+function upgrade50_verify_adminemail()
+{
+	global $output, $cache, $db, $mybb;
+
+	$output->print_header("Verifying Admin Email");
+	if(empty($mybb->settings['adminemail']))
+	{
+		echo "<p>Updating admin email settings...</p>";
+		echo "<p><small>P.D: Field can not be empty</small></p>";
+		$db->update_query('settings', array('value' => $mybb->user['email']), "name='adminemail'");
+	}
+	else
+	{
+		echo "<p>Admin email verified success...</p>";		
+	}
+	$output->print_contents("<p>Click next to continue with the upgrade process.</p>");
+	$output->print_footer("50_done");
+}
```

Comment:
```suggestion
}

function upgrade50_update_adminemail()
{
	global $output, $cache, $db, $mybb;

	$output->print_header("Updating Admin Email");
	if(isset($mybb->input['adminemail']) && filter_var($mybb->input['adminemail'], FILTER_VALIDATE_EMAIL))
	{
		$db->update_query('settings', array('value' => $db->escape_string($mybb->input['adminemail'])), "name='adminemail'");
	}
	else
	{
		// I'm unsure if this would work to go back if user input is invalid..
		$output->print_contents("<p>Admin Email can not be empty, and must be a valid email address.</p>");
		$output->print_footer("50_verify_adminemail");
	}

	echo "<p>Admin email verified success...</p>";

	$output->print_contents("<p>Click next to continue with the upgrade process.</p>");
	$output->print_footer("50_done");
}
```

@euantorano @yuliu , what do you think? Does this seems like something that makes sense and could be enough?
```